crypto/ec/ec_mult.c

```diff
@@ -151,6 +370,31 @@ int ec_wNAF_mul(const EC_GROUP *group, EC_POINT *r, const BIGNUM *scalar,
                                  * precomputation is not available */
     int ret = 0;
 
+    /* Handle the common cases where the scalar is secret, enforcing a
+     * constant time scalar multiplication algorithm.
+     */
+    if ((scalar != NULL) && (num == 0)) {
+        /* In this case we want to compute scalar * GeneratorPoint:
```

Comment:
Styling nit. Coding style suggests to put opening `/*` of multi-line comment on separate line. But it refers to it as "preferred style", not required. So I suppose this remarks is a side note [that applies to multiple occurrences here]...

Reply:
Fixed in 985f307eda